app/src/main/java/nl/avans/drivioapp/MainActivity.kt

```diff
@@ -1,40 +1,42 @@
-package nl.avans.drivioapp
-
-import android.os.Bundle
-import androidx.appcompat.app.AppCompatActivity
-import androidx.fragment.app.Fragment
-import nl.avans.drivioapp.databinding.ActivityMainBinding
-
-class MainActivity : AppCompatActivity() {
-
-    private lateinit var binding: ActivityMainBinding
-
-    override fun onCreate(savedInstanceState: Bundle?) {
-        super.onCreate(savedInstanceState)
-        binding = ActivityMainBinding.inflate(layoutInflater)
-        setContentView(binding.root)
-        replaceFragment(DiscoverFragment())
-
-        binding.bottomNavigationView.setOnItemSelectedListener {
-            when (it.itemId) {
-                R.id.discover -> replaceFragment(DiscoverFragment())
-                R.id.bookings -> replaceFragment(BookingsFragment())
-                R.id.messages -> replaceFragment(MessagesFragment())
-                R.id.profile -> replaceFragment(ProfileFragment())
-
-                else -> {
-
-                }
-            }
-            true
-        }
-
-    }
-
-    private fun replaceFragment(fragment: Fragment) {
-        val fragmentManager = supportFragmentManager
-        val fragmentTransaction = fragmentManager.beginTransaction()
-        fragmentTransaction.replace(R.id.flFragment, fragment)
-        fragmentTransaction.commit()
-    }
-}
\ No newline at end of file
+package nl.avans.drivioapp
+
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import androidx.databinding.DataBindingUtil
+import androidx.navigation.NavController
+import androidx.navigation.fragment.NavHostFragment
+import androidx.navigation.ui.AppBarConfiguration
+import androidx.navigation.ui.navigateUp
+import androidx.navigation.ui.setupActionBarWithNavController
+import androidx.navigation.ui.setupWithNavController
+import nl.avans.drivioapp.databinding.ActivityMainBinding
+
+
+class MainActivity : AppCompatActivity() {
+
+    private lateinit var appBarConfiguration: AppBarConfiguration
+    private lateinit var navController: NavController
+    private lateinit var binding: ActivityMainBinding
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        binding = ActivityMainBinding.inflate(layoutInflater)
+        setContentView(binding.root)
+
+        val navHostFragment = supportFragmentManager.findFragmentById(R.id.fragmentContainerView) as NavHostFragment
+        navController = navHostFragment.navController
+
+//        appBarConfiguration = AppBarConfiguration(navController.graph)
+//        setSupportActionBar(binding.toolbar)
+//        setupActionBarWithNavController(navController, appBarConfiguration)

```

Comment:
Moeten deze comments later nog gebuikt worden of kunnen deze eruit?

Reply:
Deze kunnen er uiteindelijk uit. Deze zijn er voor een top bar. voor nu kreeg ik die niet werkend zoals ik wilde. Ik laat het even staan om er eventueel nog naar te kijken. Als we er niet meer naar gaan kijken haal ik het eruit.